docs/LOGGING.md

```diff
@@ -48,13 +48,15 @@ Upon the device restart, any unfinished temporary log files of previous life lef
 
 If the device loses network connection to the controller, the accumulated gzip log files in the directories can not be uploaded and will continue to grow. There are two problems: 1) the disk space is limited and 2) once the connection is restored there will be too many log files to upload and most of the log information is not that useful due to long time losing the connection. The device will limit the log files for N hours post the network disconnection. For example it will keep for 10 hours of log information if the network connection is lost. The implementation can be a simple tail drop of logs after the 10 hours, or it can be to recycle the most recent 5 hours logs and leave the first 5 hours logs untouched. The device can also be in the condition of the network is reachable, but the upload is not fast enough the the disk space is almost full, for example only 100 M Bytes space left. The device will either start to tail drop the logs or the recycle the log files through portion o fthe existing log files as in the network disconnection case.
 
-Once the gzip log files are uploaded to the cloud, any log entries in them will not be available on the device. For any log files are still waiting to be uploaded, they are in the '/persist/newlog/devUpload' and '/persist/newlog/appUpload' directories. EVE developers who have enabled ssh to the device for debugging purposes can look the log entries at in those directories by using "zcat" utility.
+Once the gzip log files are uploaded to the cloud, the gzip files are still available on the device in the circular buffer in /persist/newlog/keepSentQueue directory (up to the maximum of 1000 gzip files). For any log files are still waiting to be uploaded, they are in the '/persist/newlog/devUpload' and '/persist/newlog/appUpload' directories. EVE developers who have enabled ssh to the device for debugging purposes can look the log entries at in those directories by using "zcat" utility.
```

Comment:
Does keepSentQueue have a copy of all of the log files including those which live in collect and devUpload directories? Those directories are part of the circular buffer as far as I'm concerned. Thus I think the description needs to be different. Something along the lines of:
The device retains logs in the collect, devUpload, and keepSentQueue directories which together form a circular buffer where the oldest log files are dropped from keepSentQueue when the space exceeds XXX.